DMTN-107.tex

```diff
@@ -227,54 +229,60 @@ \subsection{Commissioning Data Templates}\label{ssec:potsol_comm}
+Then, after a six-week gap, the same three week survey is executed and data is processed with the Prompt pipeline to perform DIA and generate alerts (which might be distributed with high latency via non-streaming interfaces).
+If Mini-Survey 1 is executed prior to the start of operations, then multi-band templates for $\sim$1600 $\rm deg^2$ ($\sim$10\% of WFD) would already exist and could be used for alert generation in LOY1.
+\citeds{LSE-79} also described Mini-Survey 2 as $\sim$300 $\rm deg^2$ to a 10-year equivalent depth, and a $\sim$100 $\rm deg^2$ 20-year Depth Test as part of Early Science Verification.
+Additional sky area would be observed as a part of the other Key Performance Metrics during Early Science Verification.
+{\bf The main takeaway is that \citeds{LSE-79} described that {\it at least} $\sim14\%$ of the WFD main survey area could have six-filter templates at the start of LOY1.}
+
+The ability to generate six-filter templates and distribute alerts during LOY1 over $14\%$ of the WFD area would enables some science, but would not maximize it under this Tech Note's working definition of maximizing the number of alerts for unique objects in LOY1.
+There are two options for increasing this area -- and thus the number of LOY1 alerts -- prior to the start of operations: (1) dedicate commissioning time to a template-building survey, or (2) create a template-building program for the observation scheduler which can run as "filler" during commissioning.
+
+{\bf (1) The Feasibility of a Dedicated Template-Building Survey Prior to Operations}\\
+Significantly increasing the sky area with templates by observing additional areas during the already very busy commissioning phase is probably not be feasible.
+%For example, while it might be possible to cover $\sim50\%$ of the WFD area with one image per filter per field in $\sim$6 photometric nights worth of observing, single-image depth is insufficient for templates (\S~\ref{ssec:req_temp} and \ref{ssec:potsol_imgimg}).
+For example, it might be possible to cover $\sim$50\% of the WFD area with $3$ images per field in a single filter in $\sim$3 photometric nights, which should be sufficient to build template images that meet the noise contribution requirement\ossreq{0158}\reqparam{templateNoiseLevelY1} for LSST year 1 (\S~\ref{ssec:req_temp}).
+However, that is a very idealized minimum time estimate, which assumes 1000 visits per night and does not account for weather or dithering, factors which would $\sim$double the time estimate.
+Such a survey dedicated to template-building would not necessarily serve an independent commissioning-related purpose that would make it worth spending commissioning time on -- it remains the purview of the commissioning and operations teams whether LOY1 alert production is best served by building templates or ensuring a fully-commissioned system.
```

Comment:
" Such a survey dedicated to template-building would not necessarily serve an independent commissioning-related purpose  -- it remains the purview of the commissioning and operations teams whether to carry out a dedicated template-building survey prior to operations"

I'd rather not  present this as an either/or 'fully commissioned system' vs maximising the number of template images.

Reply:
I just removed " -- it remains the purview of the commissioning and operations teams whether LOY1 alert production is best served by building templates or ensuring a fully-commissioned system." because that statement is kind of stepping out of the bounds of a DMTN anyway...
Now it just says "Furthermore, such a survey dedicated to template-building might not serve an independent commissioning-related purpose that would make it worth spending (potentially limited) commissioning time on, and priorities would need to be considered."